fleet/lib/rook-ceph-cluster/values.yaml

```diff
@@ -117,17 +117,17 @@ cephClusterSpec:
         memory: 512Mi
     mon:
       limits:
-        cpu: "2"
+        cpu: "1"
         memory: 2Gi
       requests:
-        cpu: "1"
+        cpu: "500"
         memory: 1Gi
     osd:
       limits:
-        cpu: "2"
```

Comment:
Please leave the osd limit at 2.  2 is probably on the low side.

Reply:
roger that

Reply:
Please leave the osd limit at `2`.

Reply:
didnt pushed last changes do'h